src/pepr/operator/controllers/keycloak/client-sync.ts

```diff
@@ -217,7 +213,7 @@ async function apiCall(sso: Partial<Sso>, method = "POST", authToken = "") {
   }
 
   // Remove the body for DELETE requests
-  if (method === "DELETE") {
+  if (method === "DELETE" || method === "GET") {
```

Comment:
Ditto to above comment, don't think we need this conditional since we aren't using GET anymore? Unless there was a reason to leave this around... _in case_?

Reply:
I'm going to leave it just in case.